### DISCUSSION TITLE
PyCharm or VSCode on Windows

### DISCUSSION THREAD

**Discussion by fumitoh** [actuarialopensource/website#26] [General]
I've been using PyCharm for a very long time. It's good, but as I undestand, VSCode is also a popular choice for Python development. What advantages does VSCode have over PyCharm?

**Comment by pdavidsonFIA** [comment]
I honestly am a big fan of PyCharm. For a while I was using the pro version at work which had the datagrip integration, so browsing databases was really easy.
Saying that I do still use jupyter lab outside of pycharm (even though you can run notebooks in pycharm too).

**Reply by fumitoh** [reply]
I also use PyCharm as my main IDE, but since VS code is gaining popularity, I started using it too.  I still find myself more productive with PyCharm.
I haven't used github copilot yet, but I've heard it's quite amazing. It seems to integrate with both VS code and PyCharm.
